packages/create-svelte-ux/templates/starter/package.json

```diff
@@ -16,31 +16,31 @@
     "test:unit": "vitest"
   },
   "devDependencies": {
-    "@mdi/js": "^7.3.67",
-    "@playwright/test": "^1.28.1",
-    "@sveltejs/adapter-auto": "^3.0.0",
-    "@sveltejs/kit": "^2.0.0",
-    "@sveltejs/vite-plugin-svelte": "^3.0.0",
-    "@typescript-eslint/eslint-plugin": "^7.0.0",
-    "@typescript-eslint/parser": "^7.0.0",
-    "autoprefixer": "^10.4.14",
-    "eslint": "^9.0.0",
-    "eslint-config-prettier": "^9.0.0",
-    "eslint-plugin-svelte": "^2.30.0",
-    "postcss": "^8.4.24",
-    "postcss-load-config": "^5.0.0",
-    "prettier": "^3.0.0",
-    "prettier-plugin-svelte": "^3.0.0",
-    "svelte": "^4.0.5",
-    "svelte-check": "^3.4.3",
+    "@mdi/js": "^7.4.47",
+    "@playwright/test": "^1.44.1",
+    "@sveltejs/adapter-auto": "^3.2.1",
+    "@sveltejs/kit": "^2.5.10",
+    "@sveltejs/vite-plugin-svelte": "^3.1.1",
+    "@typescript-eslint/eslint-plugin": "^7.11.0",
+    "@typescript-eslint/parser": "^7.11.0",
+    "autoprefixer": "^10.4.19",
+    "eslint": "^8.57.0",
```

Comment:
Eslint is the only one down to a major 8. It's a conflicting dep in `"@typescript-eslint/parser"` and a major eslint `9` was running into not only warnings but errors when trying to use the updated templates with npm.

Reply:
Thanks @ttytm.  I personally find eslint more of a pain than helpful most of the time (I don't use it in Svelte UX [directly](https://github.com/techniq/svelte-ux/blob/main/packages/svelte-ux/package.json) :).  No worries if it lags.

Reply:
> I personally find eslint more of a pain than helpful most of the time. 

Same experience :see_no_evil: